community/cloud-foundation/templates/healthcheck/README.md

```diff
@@ -0,0 +1,80 @@
+# Healthcheck
+
+This template creates a Compute Engine healthcheck.
```

Comment:
Instead of "Compute Engine" lets use "instance" or "load balancer" health check

Reply:
Changed